contracts/ChainlinkPriceFeedV3.sol

```diff
@@ -82,7 +82,13 @@ contract ChainlinkPriceFeedV3 is IChainlinkPriceFeedV3, IPriceFeedUpdate, BlockC
     }
 
     function isTimedOut() external view override returns (bool) {
-        return _lastValidTimestamp > 0 && _lastValidTimestamp.add(_timeout) < _blockTimestamp();
+        (, uint256 lastestValidTimestamp) = _getCachePrice();
```

Comment:
add comment
// To fetch the latest price to prevent the cached _lastValidTimestamp is outdated.

Reply:
just found that this logic is still incorrect. fixed in 87802cb